### PR TITLE
init lib with countryCode as paramater

### PR DESCRIPTION
**Pull request by id0Sch**
Hi,
we use this lib a-lot and we implemented a simple solution to change the store that we connect to.
i'm submitting this PR so the lib could improve and as a hope that we would not have to depend on a forked version of the lib and use the original!

thanks for this cool lib,

### CODE REVIEW

**Comment by id0Sch** [comment]
@dweinstein, how do i make the tests pass in travis?

**Comment by dweinstein** [comment]
hey @id0Sch thanks for the contribution. I have a couple of things to consider:

I'm a little concerned about the backward incompatible change of the arguments. In fact as we continue to add new parameters to the API's "constructor" I sometimes wish I had used an object which might provide greater flexibility on the argument order in general.

Two ideas come to mind:
- export a function that allows you to change the country code that is used by the API, rather than supplying it at construction time. Something like

```
api = Api(...)
api.setCountryCode("us")
...
api.doStuff()
```

- Bite the bullet and change the public interface of the API constructor to accept the argument and everyone will have to migrate to using the new interface

- Create a new constructor that maintains backward compatibility but allows constructing an API with the country set. Something like:

```
api.withCountryCode(<normal args>, ... country) // returns a Google API with the country code set
```

where anyone that uses api(<original args>) spawns the old-style api without breaking compat.

Long story short I'm using your PR as an excuse to consider cleaning things up a little bit.

**Comment by dweinstein** [comment]
Tthe quickest fix would be to put the countryCode at the end of the existing parameter list and that allows you to use it without breaking anyone else. I'm not sure I like it but it's the easiest for sure.

**Comment by dweinstein** [comment]
@id0Sch also I would prefer to split this PR into two separate PRs. one for the countryCode and another for the downloadInfo. I'm still wrapping my head around what functionality your after with "complete" download info as I believe we're already doing this here: https://github.com/dweinstein/node-google-play/blob/master/lib/api.js#L361

**Comment by id0Sch** [comment]
`Bite the bullet and make the constructor argument an object rather than the parameters`
i can make it so that if the first argument is an object, i disregard the rest of them and assume that it contains the needed params.

```
also I would prefer to split this PR into two separate PRs.
one for the countryCode and another for the downloadInfo -- assuming its even needed.
```
i too wanted to split it to two but for some reason github merged them back , i'll try again.

```
 I'm still wrapping my head around what functionality you're after with
"complete" download info as I believe we're already doing this here:
```
about the downloadInfo, it returns an object that is different than the one that my new function returns.
we use it because we optimized the download process to our needs and still want to use your lib for the processing of the cookie, setting of the headers and such.
i tried not to break the current structure but to add new functionality.

**Comment by id0Sch** [comment]
```Long story short I'm using your PR as an excuse to consider cleaning things up a little bit.```
i would be happy to help you if it's needed.
i would start by replacing the test interface to Mocha, tape is a pain....
but seriously, great job on the package, we rely on it so much, that this protobuf change shut us down for 1 day

**Comment by id0Sch** [comment]
anyways im opening two PR'S and we should close this one
